cluster/update-storage-objects.sh

```diff
@@ -59,7 +61,12 @@ for resource in "${resources[@]}"
 do
   for namespace in "${namespaces[@]}"
   do
+    # If get fails, assume it's because the resource hasn't been installed in the apiserver.
+    # TODO hopefully we can remove this once we use dynamic discovery of gettable/updateable
+    # resources.
+    set +e
```

Comment:
I think it will be good to check that we got at least one resource.
I want to prevent the situation where `kubectl get` failed for some reason for all resources and we think that this script passed even though it didnt do anything.